src/train/extractor/extractor.py

```diff
@@ -69,27 +70,43 @@ def extract(self, query_results, energy_components, feature_group, energy_source
         # 1. compute energy different per timestamp and concat all energy component and unit
         power_data = self.get_power_data(query_results, energy_components, energy_source)
         if power_data is None:
-            return None, None, None
+            return None, None, None, None
         power_data = drop_zero_column(power_data, power_data.columns)
         power_columns = power_data.columns
         features = FeatureGroups[FeatureGroup[feature_group]]
```

Comment:
```suggestion
fg = FeatureGroup[feature_group]
features = FeatureGroups[fg] <--  ACCELERATE_FEATURES  must be = ["accelerator_intel_qat_latency", "accelerator_intel_qat_compress", "accelerator_intel_qat"_decompress"]

if node_level == True and fg == FeatureGroup.AcceleratorOnly:
     feature_data = self.get_accelerator_feature_data(query_results, workload_features) <-- reset_index inside this function
else:
     feature_data = self.get_workload_feature_data(query_results, workload_features)

if feature_data is None:
   return None, None, None
  

```

Reply:
Thanks for your comments. How about not judging whether node_ is equal to true now? 
If node_level == False, get_workload_feature_data() will handle accelerator_features, or directly return None, None, None，which may cause other errors.

Reply:
You are right. We must consider the case that node_level is False but AcceleratorOnly is set. 
We should return None because AcceleratorOnly is not supported for the container level for now. 
Please add that case.